lib/rubyvm/debug_inspector.rb

```diff
@@ -0,0 +1,6 @@
+require 'rbconfig'
+dlext = RbConfig::CONFIG['DLEXT']
+begin
+  require "rubyvm/debug_inspector.#{dlext}"
+rescue LoadError
```

Comment:
This PR might change things a bit:
https://github.com/banister/binding_of_caller/blob/master/lib/binding_of_caller/mri2.rb

If you require 'binding_of_caller/mri2' on 1.9 you'd get a NameError on master, but on this branch it will be `NoMethodError (private method)` for `RubyVM::DebugInspector.open`.